napari/_qt/qt_main_window.py

```diff
@@ -204,6 +215,46 @@ def _add_plugins_menu(self):
         report_plugin_action.triggered.connect(self._show_plugin_err_reporter)
         self.plugins_menu.addAction(report_plugin_action)
 
+    def _show_plugin_list(self):
+        """Show dialog with a table of installed plugins and metadata."""
+        from ..plugins import plugin_manager
+
+        dialog = QDialog(self._qt_window)
+        dialog.setMaximumHeight(800)
+        dialog.setMaximumWidth(1280)
+        layout = QVBoxLayout()
+        title = QLabel("Installed Plugins")
```

Comment:
Maybe a comment for the future - could we add a search bar on the top? Does Qt have built in "search" functionality that we can leverage? Or would we need to build ourselves? Not a priority now, but curious